system/types/credentials.go

```diff
@@ -36,10 +36,9 @@ func (u *Credentials) Valid() bool {
 }
 
 // CompareHashAndPassword returns first valid credentials with matching hash
```

Comment:
with introduction of the validOnly the description of the function is no longer correct